<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{border-image: url(:/new/prefix1/pic/city.jpg);}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="next">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>220</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>下一页</string>
    </property>
   </widget>
   <widget class="QLabel" name="date">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>日期:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="previous">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>上一页</string>
    </property>
   </widget>
   <widget class="QLabel" name="weather">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>天气:</string>
    </property>
   </widget>
   <widget class="QLabel" name="weatherInfo">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>70</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>暴雨</string>
    </property>
   </widget>
   <widget class="QLabel" name="time">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>时间:</string>
    </property>
   </widget>
   <widget class="QWidget" name="W2" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>781</width>
      <height>201</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QPushButton" name="B1">
     <property name="geometry">
      <rect>
       <x>41</x>
       <y>18</y>
       <width>93</width>
       <height>171</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>B1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="B2">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>18</y>
       <width>93</width>
       <height>171</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>B2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="B3">
     <property name="geometry">
      <rect>
       <x>319</x>
       <y>18</y>
       <width>93</width>
       <height>171</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>B3</string>
     </property>
    </widget>
    <widget class="QPushButton" name="B5">
     <property name="geometry">
      <rect>
       <x>597</x>
       <y>21</y>
       <width>93</width>
       <height>171</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>B5</string>
     </property>
    </widget>
    <widget class="QPushButton" name="B4">
     <property name="geometry">
      <rect>
       <x>458</x>
       <y>21</y>
       <width>93</width>
       <height>171</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>B4</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="W1" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>781</width>
      <height>201</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QPushButton" name="A1">
     <property name="geometry">
      <rect>
       <x>41</x>
       <y>18</y>
       <width>93</width>
       <height>171</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>A1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="A2">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>18</y>
       <width>93</width>
       <height>171</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>A2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="A3">
     <property name="geometry">
      <rect>
       <x>319</x>
       <y>18</y>
       <width>93</width>
       <height>171</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>A3</string>
     </property>
    </widget>
    <widget class="QPushButton" name="A5">
     <property name="geometry">
      <rect>
       <x>597</x>
       <y>21</y>
       <width>93</width>
       <height>171</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>A5</string>
     </property>
    </widget>
    <widget class="QPushButton" name="A4">
     <property name="geometry">
      <rect>
       <x>458</x>
       <y>21</y>
       <width>93</width>
       <height>171</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>A4</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="W3" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>781</width>
      <height>201</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QPushButton" name="C1">
     <property name="geometry">
      <rect>
       <x>41</x>
       <y>18</y>
       <width>93</width>
       <height>171</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>C1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="C2">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>18</y>
       <width>93</width>
       <height>171</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>C2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="C3">
     <property name="geometry">
      <rect>
       <x>319</x>
       <y>18</y>
       <width>93</width>
       <height>171</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>C3</string>
     </property>
    </widget>
    <widget class="QPushButton" name="C5">
     <property name="geometry">
      <rect>
       <x>597</x>
       <y>21</y>
       <width>93</width>
       <height>171</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>C5</string>
     </property>
    </widget>
    <widget class="QPushButton" name="C4">
     <property name="geometry">
      <rect>
       <x>458</x>
       <y>21</y>
       <width>93</width>
       <height>171</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>C4</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="residual">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>20</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>剩余车位:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="previous_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>我是管理员</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="timeInfo">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>130</y>
      <width>151</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 170, 255);</string>
    </property>
    <property name="digitCount">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="dateInfo">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 170, 255);</string>
    </property>
    <property name="digitCount">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="residualInfo">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>30</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 170, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="pic">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>110</y>
      <width>160</width>
      <height>120</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>160</width>
      <height>120</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>160</width>
      <height>120</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>W3</zorder>
   <zorder>W2</zorder>
   <zorder>W1</zorder>
   <zorder>next</zorder>
   <zorder>date</zorder>
   <zorder>previous</zorder>
   <zorder>weather</zorder>
   <zorder>weatherInfo</zorder>
   <zorder>time</zorder>
   <zorder>residual</zorder>
   <zorder>previous_2</zorder>
   <zorder>timeInfo</zorder>
   <zorder>dateInfo</zorder>
   <zorder>residualInfo</zorder>
   <zorder>pic</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
